### PR TITLE
Explicitly feature works in static feeds

### PR DESCRIPTION
**Pull request by courte**
This branch fixes a lot of issues holding back static feeds, including (but not limited to):
  - Avoiding unions, which perform poorly in this cases, preferring to extract objects instead.
  - Fixing the annotator to work with URLs that have an ending slash.
  - Using xml -- which S3 will host -- instead of opds, which it doesn't know what to do with.
  - Extracting categories from yaml (because I had some time on my hands trying to figure out the database nonsense and it makes tiered lane structures slightly easier to grok).
  - Featuring works based on human additions to the CSV.
  - Denying StaticFeedBaseLane its previous, more regal moniker (`IdentifiersLane`) and instead casting it into the street like the groveling, script-specific pauper that it truly is.

Long story short: static feeds can be generated and uploaded and viewed in clients now.

There is still a query issue somewhere related to `Works` being pulled from the database without all of their `LicensePools` (so indecent!), but I will fix that in a future branch.

### CODE REVIEW

File: lanes.py [hunk 1]
```diff
@@ -7,17 +8,19 @@
...
 
-class IdentifiersLane(QueryGeneratedLane):
 
-    def __init__(self, _db, identifiers, lane_name, **kwargs):
+class StaticFeedBaseLane(QueryGeneratedLane):
+
+    def __init__(self, _db, identifiers, lane_name, featured=None, **kwargs):
         if not identifiers:
             raise ValueError(
                 "IdentifierGeneratedLane can't be created without identifiers"
```
Comment (leonardr) [inline lanes.py]: You should change the name of the class here.

**Comment by leonardr** [comment]
:+1:
